lib/MagicPen.js

```diff
@@ -24,7 +32,12 @@ function MagicPen(options) {
     this.output = [[]];
     this.styles = Object.create(null);
     this.installedPlugins = [];
-    this._themes = {};
+    // Ready to be cloned individually:
+    this._themes = {
+        html: { styles: {} },
+        ansi: { styles: {} },
+        text: { styles: {} }
+    };
```

Comment:
I on my phone, so it is a bit hard to get an overview. But doesn't this count on the number of formats to be a closed set? What about the colored console format?